samples/python/src/getting_started.py

```diff
@@ -13,11 +17,15 @@
 async def run():
     logger.info("Getting started -> started")
 
-    logger.info("Open Pool Ledger")
     pool_name = 'pool1'
+    logger.info("Open Pool Ledger: {}".format(pool_name))
     pool_genesis_txn_path = get_pool_genesis_txn_path(pool_name)
     pool_config = json.dumps({"genesis_txn": str(pool_genesis_txn_path)})
-    await pool.create_pool_ledger_config(pool_name, pool_config)
+    try:
+        await pool.create_pool_ledger_config(pool_name, pool_config)
+    except IndyError as ex:
+        if ex.error_code == ErrorCode.PoolLedgerConfigAlreadyExistsError:
+            pass
```

Comment:
I'm not sure that try/except is a good decision here. You added it for getting_started.py but did not for anoncreds, crypto, ledger. 
Ideally, samples must run from a clean folder and clean everything after them. 
I think it would be better to add cleaning of .indy_client folder before each sample.

Reply:
Ledger, anoncreds and crypto had the the code to delete wallets